spec/02-integration/11-dbless/04-pagination_spec.lua

```diff
@@ -45,5 +50,21 @@ describe("dbless pagination #off", function()
     assert.res_status(201, res)
     admin_client:close()
 
+    local res = admin_client:get("/routes/my-route-1")
+    print(res:read_body())
+    print("-------")
+
+    -- check routes number with :page() API
+    --local res, err  = client:get("/1", {})
+
+    local res = assert(client:send {
+      method = "GET",
+      path = "/1",
+    })
+    --print(require("inspect")(res))
+    local resbody = res:read_body()
+    print"--------"
+    print(resbody)
+    --assert.response(res).has.header("X-rows-number", "test")
```

Comment:
alwasy 404???

```
.......  RUN 36: dbless pagination #off Routes
{"sources":null,"snis":null,"headers":null,"methods":null,"path_handling":"v0","protocols":["http","https"],"created_at":1732593028,"updated_at":1732593028,"strip_path":true,"regex_priority":0,"destinations":null,"preserve_host":false,"service":{"id":"2ead4a03-02e4-5d5d-bf3e-afe3aa5bf683"},"request_buffering":true,"response_buffering":true,"https_redirect_status_code":426,"name":"my-route-1","id":"ed2fc784-0fcf-5bcb-b724-174dab625100","paths":["/1"],"hosts":null,"tags":null}
-------
--------
{
  "message":"no Route matched with those values",
  "request_id":"e44b1e47c11c9e1bc7bce6794ec92135"
```

Reply:
If we user bp.routes:insert, it worked, posting config to admin api failed to insert route